layer/mounted_layer.go

```diff
@@ -155,6 +156,12 @@ func (rl *referencedRWLayer) Mount(mountLabel string) (string, error) {
 
 	if rl.activityCount > 0 {
 		rl.activityCount++
+		path, err := rl.mountedLayer.layerStore.GetMountPathFromDisk(rl.mountedLayer.name)
```

Comment:
No need to keep them as public methods if they are only used internally. Probably also not needed to add them to `layerStore` at all, instead it would be nice to add it as a property of `mountedLayer`. Then it is cached for remounts and loaded on startup like the rest of the properties there.